db/stop_database.go

```diff
@@ -27,20 +26,39 @@ const (
 	ServiceStopFailed
 	// ServiceStopTimedOut :: StopStatus - service stop attempt timed out
 	ServiceStopTimedOut
+	// ServiceStopFailed_IsImplicit :: StopStatus - when service is running implicit to a different command
```

Comment:
where are these used?

Reply:
This got left over from an earlier iteration. Removed